api/mod/tests/integration/test_login_router.py

```diff
@@ -28,13 +24,42 @@ def test_unknown_user_returns_401(client: TestClient) -> None:
     assert response.status_code == 401
 
 
-def test_invalid_password_returns_401(
-    client: TestClient, db: Session, user: CreateUserRequest
-) -> None:
+def test_invalid_password_returns_401(client: TestClient) -> None:
     """Test POST /login raises HttpException 401"""
-    DBUser.create(db=db, user=user)
     response = client.post(
         "/login/",
         json={"email": "team@folivora.online", "password": "wrong_password"},
     )
     assert response.status_code == 401
+
+
+def test_login_token_is_usable(client: TestClient) -> None:
+    """Test POST /login"""
+    response = client.post(
+        "/login/",
+        json={"email": "team@folivora.online", "password": "password"},
+    )
+    assert response.status_code == 200
+    token = response.json()["access_token"]
+    response = client.get("/tasks", headers={"Authorization": f"Bearer {token}"})
+    assert response.status_code == 200
+
+
+def test_login_token_expires(client: TestClient, mocker: MockerFixture) -> None:
+    """Test POST /login"""
+    mocker.patch("time.time", lambda: 0)
```

Comment:
I think you dont need a lambda here. `return_value=0` works too. What do you prefer? For consistency we should change it in other places as well, if we want to keep lambda

Reply:
return_value=0 is perfect